influxdb/1.2/alpine/entrypoint.sh

```diff
@@ -1,8 +1,128 @@
-#!/bin/sh
+#!/bin/bash
 set -e
 
 if [ "${1:0:1}" = '-' ]; then
     set -- influxd "$@"
 fi
 
-exec "$@"
+AUTH_ENABLED="$INFLUXDB_HTTP_AUTH_ENABLED"
+
+if [ -z "$AUTH_ENABLED" ]; then
+	AUTH_ENABLED="$(grep -iE '^\s*auth-enabled\s*=\s*true' /etc/influxdb/influxdb.conf | grep -io 'true' | cat)"
```

Comment:
The configuration file is not getting rewritten, but just read to check if auth-enabled is set to true.